### ISSUE TITLE
Format string error displayed when invalid numeric parameter value entered

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/pashi#24]
If an invalid numeric parameter value is entered then the error message displayed is:

```text
Error: Format '%0:s' invalid or incompatible with argument
```

Instead of the expected error message about an invalid number being entered.

For example, this message gets displayed when the parameter `--line-number-width 20` is entered, where `20` is an invalid value for the `--line-number-width` parameter.

**Comment by delphidabbler** [comment]
Need to change https://github.com/delphidabbler/pashi/blob/639e9a8134ecf3ae1fcae8263e67f6f580742533/Src/UParams.pas#L448

To

```pascal
sOutOfRange = 'Parameter for %s must be in range '; 
```

And change
https://github.com/delphidabbler/pashi/blob/639e9a8134ecf3ae1fcae8263e67f6f580742533/Src/UParams.pas#L454

To

```pascal
raise ECommandError.Create(
  Cmd.Name, 
  sOutOfRange + Format('%0:d..%1:d', [Lo, Hi])
);

**Comment by delphidabbler** [comment]
Fixed at commit 20893cd
